src/language-server/universal-data-definition-language-formatter.ts

```diff
@@ -21,22 +21,29 @@ export class UniversalDataDefinitionLanguageFormatter extends AbstractFormatter
         close.surround(Formatting.noSpace()).prepend(Formatting.newLine({allowMore: true})).append(Formatting.newLine({allowMore: true}));
     }
 
+    protected formatNode(node: AstNode): void {
+        const formatter = this.getNodeFormatter(node);
+        formatter.property('name').prepend(Formatting.newLine()).surround(Formatting.oneSpace({allowMore: true}))
+    }
+
     protected format(node: AstNode): void {        
         // This method is called for every AstNode in a document
         if (ast.isDataModel(node)) {
             this.formatDataModel(node);
         }
-        if(ast.isConceptualAssociation(node)){            
-            this.formatConceptualAssociation(node)
-        }
+        // test passes with this commetned check
```

Comment:
The test you have defined does not include a ConceptualAssociation, so commenting this method out will have no effect on test success/ failure.